### DISCUSSION TITLE
Python SDK

### DISCUSSION THREAD

**Discussion by anorneto** [openfga/community#50] [Q&A]
Hello everyone,

I saw the python-sdk is on the roadmap. Is there any release date planned? Im planning on using OpenFGA for flask apis :)

Thanks!

**Comment by rhamzeh** [comment]
Hi @anorneto!

**Update**: 

The Python SDK is out now, you can find the repo on [GitHub](https://github.com/openfga/python-sdk) and the package on [PyPi](https://pypi.org/project/openfga-sdk/). Try it out and let us know your feedback!

--

**Earlier answer**:

Yes! We are working on publishing it very soon, hopefully next week!

If you can't wait until then, you are also able to build it yourself using the [SDK generator](https://github.com/openfga/sdk-generator).

1. Clone the SDK Generator
2. run `make build-client-python`
3. You will find the generated Python SDK in the `client/fga-python-sdk` subdirectory

In order to build it, you will need the following tools to be available on your machine:
- `make`
- `docker`
- `sed`
- `cat`
- `curl`

**Comment by aaguiarz** [comment]
The Python SDK is already available https://github.com/openfga/python-sdk
